### PR TITLE
Fix the error of kms_transformation_test about integration test

### PR DESCRIPTION
**Pull request by yangjunmyfm192085**
Signed-off-by: JunYang <yang.jun22@zte.com.cn>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
@kubernetes/sig-auth-pr-reviews

File: test/integration/master/kms_transformation_test.go [hunk 1]
```diff
@@ -257,7 +257,6 @@ resources:
 
 func mustBeHealthy(t *testing.T, checkName string, clientConfig *rest.Config) {
 	t.Helper()
-	var restErr error
```
Comment (liggitt) [inline test/integration/master/kms_transformation_test.go:260]: I think this was trying to capture the inner error to give more useful debugging messages if the poll timed out. Suggest assigning the inner err to restErr

Comment (yangjunmyfm192085) [reply test/integration/master/kms_transformation_test.go]: Thank you,I adjusted the wrong order of printing and clear description added.
Is it ok?

File: test/integration/master/kms_transformation_test.go [hunk 2]
```diff
@@ -267,7 +267,7 @@ func mustBeHealthy(t *testing.T, checkName string, clientConfig *rest.Config) {
 	})
 
 	if pollErr == wait.ErrWaitTimeout {
```
Comment (liggitt) [inline test/integration/master/kms_transformation_test.go:269]: I do think we still need to assign `restErr = err` inside the loop above

File: test/integration/master/kms_transformation_test.go [hunk 3]
```diff
@@ -283,7 +283,7 @@ func mustBeUnHealthy(t *testing.T, checkName string, clientConfig *rest.Config)
...
```
Comment (liggitt) [inline test/integration/master/kms_transformation_test.go:285]: I do think we still need to assign `restErr = err` inside the loop above

Comment (yangjunmyfm192085) [reply test/integration/master/kms_transformation_test.go:287]: yeah，I agree，now i fixed it.

File: test/integration/master/kms_transformation_test.go (hunk 2 above)
Comment (yangjunmyfm192085) [reply test/integration/master/kms_transformation_test.go:270]: yeah，I agree，now i fixed it.

**Pull request merged**